Guide.docc/MigrationGuide.md

```diff
@@ -14,27 +14,27 @@
 
 Swift's concurrency system, introduced in [Swift 5.5](https://www.swift.org/blog/swift-5.5-released/),
 makes asynchronous and parallel code easier to write and understand.
-With the Swift 6 language mode, the compiler can now also
-guarantee that concurrent programs are data-race-free.
-When this mode is enabled, the compiler safety checks that were
-optional become required.
+With the Swift 6 language mode, the compiler can now 
+guarantee that concurrent programs are free of data races.
+With this mode enabled, compiler safety checks that were
```

Comment:
I just realized this suggested change has two sentences in a row beginning with 'With' which is probably not a great thing.

I think that just starting this sentence with "When enabled, compiler safety checks..." it is clear enough that Swift 6 language mode is being referenced from the previous sentence.